models/posts.js

```diff
@@ -0,0 +1,26 @@
+const mongoose = require('mongoose');
+
+const databaseURL = 'mongodb+srv://axel:axel123@obscuracluster-2swgt.mongodb.net/test?retryWrites=true&w=majority'; 
+const options = { useNewUrlParser: true,
+  useUnifiedTopology: true,
+  useFindAndModify: false };
+
+mongoose.connect(databaseURL, options);
+
+const postSchema = new mongoose.Schema({
+    author_id: {type: mongoose.Schema.Types.ObjectId, ref: 'users', required: [true, "no author id provided"]}, 
+    author: {type: String, required: [true, "No author name provided"]},
```

Comment:
I think the 'author_id' is confusing and the 'author' field is redundant.

1. author_id references to 'users'
2. When you populate() author_id, it's replaced with the mongoose document that corresponds to that author_id

Suggestion:
Remove the original 'author' and 'author_id' field and replace it with:
`author: {type: mongoose.Schema.Types.ObjectId, ref: 'users', required: [true, "no author id provided"]}`